tests/test_bin_trie.py

```diff
@@ -100,3 +100,15 @@ def test_bin_trie_invalid_key(invalide_key, if_error):
         previous_root_hash = trie.root_hash
         trie.delete(invalide_key)
         assert previous_root_hash == trie.root_hash
+
+
+@given(keys=st.lists(st.binary(min_size=32, max_size=32), min_size=100, max_size=100, unique=True),
+       chosen_numbers=st.lists(st.integers(min_value=0, max_value=99), min_size=50, max_size=50))
+@settings(max_examples=10)
+def test_bin_trie_update_value(keys, chosen_numbers):
```

Comment:
This test has no assertions. I'm confused as to what it's doing.  If there are no reasonable assertions to make, can you update the test with a docstring explaining what it's doing and why there are no assertions.

Reply:
Sorry about the confusion. It's a basic test to see if `BinaryTrie.update` is working as expected. Just making sure that basic functionality like `update` works correctly.

Added doctoring for explanation and conditions check.